src/plugins/usage_collection/kibana.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "usageCollection",
-  "configPath": ["usageCollection"],
+  "configPath": ["usageCollection.uiMetric"],
```

Comment:
Doing this because the config `"usageCollection"` is already registered by core. We are extending only the config re `uiMetric` for the public-end code